sympy/physics/mechanics/actuator.py

```diff
@@ -990,3 +991,196 @@ def __repr__(self):
         return (f"{self.__class__.__name__}("
                 f"{self.linear_stiffness}, {self.nonlinear_stiffness}, {self.pathway}, "
                 f"equilibrium_length={self.equilibrium_length})")
+
+class CoulombKineticFriction(ForceActuator):
+    r"""Coulomb kinetic friction with Stribeck and viscous effects.
+
+    Explanation
+    ===========
+
+    This represents a Coulomb kinetic friction with the Stribeck and viscous effect,
+    described by the function:
+
+    .. math::
+        F = (\mu_k f_n + (\mu_s - \mu_k) f_n e^{-(\frac{v}{v_s})^2}) \text{sign}(v) + \sigma  v
```

Comment:
You have to pick two primary sign conventions that amount to: 1is the friction force vector defined in the same or opposite direction as the velocity.

If I look at the viscous term: $F = \sigma v$ then it seems your sign convention is that F is defined opposite of v. Then friction opposes the motion.

If I look at the Coulomb term: $\mu_k f_n \sgn v$ then F is defined opposite of v.

So viscous and Coulomb follow the same convention.

You need to declare somewhere in the docstring that you are choosing this convention.

Note that you can choose a sign convention for v1, v2, F1 and F2 resulting in many permutations of possible sign conventions:

```
   ---> v1                -----> v2
   |                      |
P1 O<-----F1     F2------>O P2
```

I think that we define such a sign convention in the Pathway base and Actuator base, so we should make sure we are following that convention here.

Reply:
See the assumptions under notes. Normal forces is a non-negative scalar and friction force is opposite to the velocity is mentioned over there.